assets/javascripts/discourse/components/post-voting-comment-actions.js

```diff
@@ -26,4 +56,17 @@ export default class PostVotingCommentActions extends Component {
       },
     });
   }
+
+  @action
+  showFlag() {
+    this.comment.availableFlags = this.comment.available_flags;
```

Comment:
I don't think this is necessary
```suggestion
```

Reply:
This is related to the reviewables pipeline too, without this it is unable to identify the proper attributeit tries to use `availableFlags` but it only has `available_flags`